src/ploomber/cli/install.py

```diff
@@ -8,13 +8,17 @@
 
 from ploomber.io._commander import Commander
 
+from ploomber.telemetry import telemetry
+import datetime
+
 _SETUP_PY = 'setup.py'
 
 _REQS_LOCK_TXT = 'requirements.lock.txt'
 _REQS_TXT = 'requirements.txt'
 
 _ENV_YML = 'environment.yml'
 _ENV_LOCK_YML = 'environment.lock.yml'
+start_time = datetime.datetime.now()
```

Comment:
this should be inside the function call, otherwise it's going to always return the time when the module is imported